### DISCUSSION TITLE
right tool for offline analysis

### DISCUSSION THREAD

**Discussion by muzzol** [osalvador/ReplicaDB#72] [Q&A] [answered]
kaixo!

I need to dump a database and work with an offline copy to perform some analytics.
Is ReplicaDB the right tool to do this?

I don't need any kind of system translation as source and destination will be the same engine, but maybe some data anonymization/obfuscation.

**Comment by osalvador** [comment, marked as answer]
Hi @muzzol,

Yes, ReplicaDB is the right tool for this kind of job. But remember that ReplicaDB doesn't change your data, just copy it from a source database to the sink or destination.

I always recommend using the [setup wizard](https://osalvador.github.io/ReplicaDB/wizard/index.html) to simplify the process.

Greetings!
